app/main/views/feedback.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+from werkzeug.exceptions import ServiceUnavailable
+from werkzeug.datastructures import MultiDict
+from werkzeug.urls import url_parse
+
+from flask import current_app, request, redirect, flash, Markup
+
+from .. import main
+
+
+@main.route('/feedback', methods=["POST"])
+def send_feedback():
+    feedback_config = current_app.config['DM_FEEDBACK_FORM']
+    form_data = MultiDict()
+    for field, google_form_field in feedback_config['fields'].items():
+        form_data.setlist(google_form_field, request.form.getlist(field))
+
+    result = requests.post(feedback_config['uri'], list(form_data.items(multi=True)))
+    if result.status_code != 200:
+        raise ServiceUnavailable('Google forms submission problem (status %d)'.format(result.status_code))
```

Comment:
Are we definitely happy using Google forms as the submission method?

Either way I'd probably take Google out of the error message and have it just say something more generic like `There was a problem submitting your feedback. Please try again later.` (i.e. not calling out Google).

Reply:
Yeah I think Google Forms is the MVP for this. Once we understand volumes and who tends to need to deal with these things then we look at alternatives.

But good point re the message.

- [ ] update error message